### ISSUE TITLE
Please do not update extensions automatically.

### ISSUE THREAD

**Issue by HT-7** [kiwibrowser/src.next#819]
TLDR:
Updates can and frequently *do* break things. Therefore, extensions should never update without user consent.

----

The "click&clean" extension spontaneously stopped working one day. Only a useless blank page appears when I open it from the side menu.

Then I looked into its page source code at `view-source:chrome-extension://ghgabhipcejejjmhhchfonmamedcbeod/emenu712.html`:

The first line: 

> Clean for Google Chrome - HOTCLEANER.COM - Copyright (c) 2023 Mixesoft.

***2023?!*** I installed it in 202**2**! So Kiwi must have updated this extension, and that update caused it to break.

Please do never update extensions without the consent of the user. Thank you.

Now I will have to hunt for a somewhat useful alternative and beg that it works.

This is reminiscent of the May 2019 Firefox disaster.

**Comment by d3ward** [comment]
If an extension works on Kiwi, then an update makes it stop working properly, I think it is better to contact the extension devs and ask them to check the issues. A lot of good extensions got user feedback and have fixed issues, now updates are no longer broken for them. If you don't want automatic updates, you could use the local crx-zip load of an extension and update manually when needed.
